packages/tradershub/src/flows/RealAccountSIgnup/SignupWizard/Actions/index.tsx

```diff
@@ -5,6 +5,7 @@ import { Button, Divider, useDevice } from '@deriv-com/ui';
 import { useSignupWizardContext } from '../../../../providers/SignupWizardProvider';
 
 type TActions = {
+    isSubmitBtnLoading?: boolean;
```

Comment:
[nit]: Can we fix the naming convention in the folder name? `RealAccountSIgnup` to `RealAccountSignup`

Reply:
Thanks @matin-deriv for noticing and pointing it out. Just checked and it'll be a rename in a couple of files due to imports.
As suggested by @mahdiyeh-deriv, we're going to rename the flow and related components to real account signup as its conflicting with the account signup flow namings. 
Will rename the folder with proper case there.